### DISCUSSION TITLE
Shortcut to close tab by forcefully exiting splits (implemented)

### DISCUSSION THREAD

**Discussion by lg28literconvectionmicrowaveoven** [ghostty-org/ghostty#3751] [Feature Requests, Ideas] [answered]
With there being a shortcut to open a new tab I thought it would be intuitive to have a shortcut to close a tab (just as clicking the close button on the tab bar). I just made it when I found there wasn't one. It's the `close_tab` action in my fork of Ghostty. I've also set the default keybind for it, but I haven't made it MacOS for lack of a machine to test it on.

Here's my [fork](https://github.com/lg28literconvectionmicrowaveoven/ghostty/tree/close-tab)

**Comment by mitchellh** [comment, marked as answer]
#4331 so feel free to PR if you have it.
